conga-aem-definitions/src/main/templates/aem-cms/aem-cms-publish-replicationagents.json.hbs

```diff
@@ -28,6 +28,7 @@
         "triggerReceive": "true",
         "triggerSpecific": "true",
         "noVersioning": "true",
+        "aliasUpdate": {{defaultIfEmpty this.aliasUpdate false}},
```

Comment:
i think we should enclose the boolean value in "" to be consistent with the other booleans

Reply:
done.